examples/megatron/models/vllm_policy_inference.py

```diff
@@ -138,21 +138,22 @@ def decode_internal(self, batched_outputs):
         prompt_sizes = torch.tensor([len(q) for q in no_padded_query_ids], device=all_tokens.device)
         loss_mask = get_loss_mask(all_tokens, self.tokenizer.tokenizer.eos_token_id, prompt_sizes)
         loss_mask = loss_mask.to("cpu")
+        print(f"str_outputs: {len(str_outputs)} {str_outputs}")
         return {"all_tokens": all_tokens, "str_outputs": str_outputs, "str_prompts": str_prompts,
             "no_padded_query_ids": no_padded_query_ids, "logprobs": logprobs,
             "loss_mask": loss_mask}
 
-class VLLMPolicyInferenceAsync(VLLMPolicyInference):
-    """VLLMPolicyInferenceAsync is the model for VLLMModuleV2, which uses async generate API"""
+class VLLMPolicyInferenceV2(VLLMPolicyInference):
```

Comment:
It's the same to VLLMPolicyInference, we can delete it

Reply:
Done